src/System.Memory/src/MatchingRefApiCompatBaseline.txt

```diff
@@ -0,0 +1,6 @@
+Compat issues with assembly System.Memory:
+MembersMustExist : Member 'System.MemoryExtensions.AsReadOnlyMemory<T>(System.Memory<T>)' does not exist in the implementation but it does exist in the contract.
```

Comment:
Should be fixed once https://github.com/dotnet/coreclr/commit/71aa610f8e540b082b0126921cafd040dfd9ca6d mirrors into corefx.